ui/src/entities/User.ts

```diff
@@ -41,3 +45,14 @@ export const fetchCurrentUser = async (): Promise<User> => {
 
   return data.me?.length ? data.me[0] : undefined;
 };
+
+export const sendPasswordResetOTP = async (payload: SendOTPPayload): Promise<undefined> => {
```

Comment:
We don't need to create a new interface for accepting arguments of this function. It is (and will quite likely always be) a single string. We can save ourselves some boilerplate by making it `async (email: string)` instead. 

```suggestion
export const sendPasswordResetOTP = async (email: string): Promise<undefined> => {
```